.github/workflows/test.yml

```diff
@@ -39,6 +39,9 @@ jobs:
           INFURA_PROJECT_SECRET: ${{ secrets.INFURA_PROJECT_SECRET }}
           EVM_RPC_URL: ${{ secrets.EVM_RPC_URL }}
           WALLET_PUBLIC_ADDRESS: ${{ secrets.WALLET_PUBLIC_ADDRESS }}
+          INFURA_IPFS_PROJECT_ID: ${{ secrets.INFURA_IPFS_PROJECT_ID }}
+          INFURA_IPFS_PROJECT_SECRET: ${{ secrets.INFURA_IPFS_PROJECT_SECRET }}
```

Comment:
maybe to keep it consistent "INFURA API KEY SECRET"?

Reply:
No it's important to keep a clear distinction between Web3 API and IPFS keys.

Reply:
In my opinion it should be named same like on Infura dashboard to avoid misunderstanding